exercises/practice/anagram/.meta/proof.ci.wren

```diff
@@ -0,0 +1,26 @@
+class Strings {
+  static downcase(s) {
+    return s.bytes.map { |x|
+      if ((65..90).contains(x)) x = x + 32
+      return String.fromByte(x)
+    }.join("")
+  }
+}
```

Comment:
No downcase functionality in the standard library? 😮

Reply:
No because its get hung up on whether it must be fully UTF-8 compliant (and if so what tables to use - without introducing dependencies) or whether ASCII is truly enough - and there is plenty of disagreement. :-)

`wren-console` includes `wren-essentials` which includes this functionality (in an add-on) but my assumption so far for the exercises themselves has been that they should stick to portable Wren code that could run anywhere - so in that case, no.

Reply:
Haha, I love a little bit of discussion!